torch/distributed/fsdp/_flat_param.py

```diff
@@ -708,7 +708,7 @@ def _init_flat_param_and_metadata(
             and aligned_numel > 0
             and total_numel != total_numel_without_padding
         ):
-            log.info(
+            logger.info(
```

Comment:
I think this also can be a debug level message as FlatParams is a underlying implementation details that user might not aware of